index.js

```diff
@@ -118,5 +165,13 @@ console.log('hello world')
  *
  * Example Output: 2, 10, 20
  */
+multiplyByTwo = (number) => {
```

Comment:
When you define an arrow function, the variable name it's stored in needs a keyword:

```js
const multiplyByTwo = () => {}
```